MapboxDirections/MBIntersection.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+public class Intersection: NSObject, NSSecureCoding {
+    public var inIndex: Int?
+    public var outIndex: Int?
+    public var entry: [Bool]
```

Comment:
Instead of an array of Booleans, this property should be an IndexSet called `outlets` that contains the indices of the roads through which the user can exit the intersection.